src/scenes/Event/scenes/topic.js

```diff
@@ -226,6 +227,15 @@ export default class TopicPage extends Component {
     const qtumReturnRate = totalBetAmount ? ((qtumWinnings - totalBetAmount) / totalBetAmount) * 100 : 0;
     const botReturnRate = totalVoteAmount ? ((botWinnings - totalVoteAmount) / totalVoteAmount) * 100 : 0;
     const invalidOption = localizeInvalidOption(topic.options[topic.resultIdx], intl);
+
+    const totalQtumWinningBets = topic.qtumAmount[topic.resultIdx];
+    const totalQtumBets = _.sum(topic.qtumAmount);
+    const totalLosingQtumBets = totalQtumBets - totalQtumWinningBets;
+    const losersQtumReward = totalLosingQtumBets / 100;
+    const losersAdjustedQtum = totalLosingQtumBets - losersQtumReward;
+    const qtumWon = ((resultBetAmount / totalQtumWinningBets) * losersAdjustedQtum);
```

Comment:
instead of doing that `_.isNumber(qtumWon) ? qtumWon : 0` below, can we just do
```js
const qtumWon = ((resultBetAmount / totalQtumWinningBets) * losersAdjustedQtum) || 0;
```